### ISSUE TITLE
Show search results somewhere in teacher profile

### ISSUE THREAD

**Issue by nicogarcia** [nicogarcia/classes-web-client#72]
Para mejorar la experiencia de búsqueda y que se pueda navegar mejor entre los profesores

**Comment by tomasgil123** [comment]
I agree. No lo pondria como algo prioritario, ya que, a priori, se supone que el profesor no va a estar interesado en buscar profes una vez que ya se registro (si lo hace, seria al principio, cuando esta viendo si se registra o no)

**Comment by nicogarcia** [comment]
Me refería a la búsqueda del estudiante. Igual coincido en que no es prioritario.
